hq/app/logic/SnykDisplay.scala

```diff
@@ -2,63 +2,60 @@ package logic
 
 import com.fasterxml.jackson.core.JsonParseException
 import play.api.libs.json._
-import utils.attempt.{Attempt, FailedAttempt, Failure}
+import utils.attempt.{Attempt, Failure}
 
 import scala.concurrent.ExecutionContext
 import model._
 import model.Serializers._
 
 object SnykDisplay {
 
-  def getOrganisation(s: String, requiredOrganisation: SnykOrganisationName): Attempt[SnykOrganisation] = {
-    val organisation = for {
-      orglist <- parseJsonToOrganisationList(s)
-      org <- orglist.find(_.name == requiredOrganisation.value)
-    } yield org
-    val error = parseJsonToError(s)
-    val f = Failure(s"Unable to find organisation from $s", s"Could not read Snyk response (${error.error})", 502, None, None)
-    Attempt.fromOption(organisation, FailedAttempt(f))
+  def parseOrganisations(s: String, snykGroupId: SnykGroupId)(implicit ec: ExecutionContext): Attempt[List[SnykOrganisation]] = {
```

Comment:
Is there a better name for `s`? Not a big deal, just hard to see what it is here from the name `s`.

Reply:
After study, perhaps something that mentions `json`?